app.js

```diff
@@ -1,9 +1,22 @@
 import express from 'express';
 import logger from 'morgan';
-import './env.js'
+import './env.js';
+import { db } from './db/dbConnection.js'
 import { router as indexRouter } from './routes/index.js';
 import { router as todoRouter } from './routes/todos.js';
 
+db.none(
```

Comment:
I suggest creating a dedicated file named `migration.js` in the `db` folder. This file will contain a method `init(db)` which will create the table. If we need to create another table, we can put all create table scripts in this file. By default, this kind of process is called migration.
I think the `init` method should have the db as parameter and the db in only created in the `app.js` file.